ArduCopter/AP_Arming.cpp

```diff
@@ -708,6 +708,11 @@ bool AP_Arming_Copter::arm_checks(AP_Arming::Method method)
         return false;
     }
 
+    // Throttle Failsafe check
+    if (copter.g.failsafe_throttle == FS_THR_DISABLED) {
+        GCS_SEND_TEXT(MAV_SEVERITY_NOTICE, "FS_THR_ENABLE is disabled");
```

Comment:
maybe reword a bit
```suggestion
        GCS_SEND_TEXT(MAV_SEVERITY_NOTICE, "Throttle failsafe is disabled");
```

Reply:
@nkruzan san.
Thanks for the comment.
I agreed.